### PR TITLE
[MDS-4805] Add update status incident

### PR DESCRIPTION
**Pull request by areyeslo**
## Objective 

[MDS-4805](https://bcmines.atlassian.net/browse/MDS-4805)

- Ability to change status of an incident in Core:
image

- Change ‘Initial Report Submitted' status to 'Written Notice Submitted’

- Add a new Reviewing Severity status

- Add a new Information Missing status

### CODE REVIEW

File: services/core-web/src/components/Forms/incidents/IncidentForm.js [hunk 1]
```diff
@@ -736,6 +769,68 @@ const renderInternalDocumentsComments = (props, isEditMode, handlers, parentHand
...
+                  props.incident?.status_code,
+                  props.incident?.update_user,
+                  props.incident?.update_timestamp,
+                  responsibleInspector,
+                  selectedStatusCode
+                )}
+              </p>
+            </Col>
+            <Col xs={24} md={6}>
+              {props.incident?.status_code && !isClosed && (
```
Comment (BenHarrington) [inline services/core-web/src/components/Forms/incidents/IncidentForm.js]: Looks like "isClosed" set above is also using props.incident?.status_code do we need this redundant check here? Just trying to keep conditionals as clean and readable as possible.

File: services/core-web/src/components/Forms/incidents/IncidentForm.js [hunk 2]
```diff
@@ -73,6 +75,37 @@ const documentColumns = [
   uploadDateColumn("upload_date"),
 ];
 
+const alertText = (status, updateUser, updateDate, responsibleInspector, selectedStatusCode) => {
+  let text = "";
+  if (selectedStatusCode === "UNR" && !responsibleInspector) {
```
Comment (JustinMacaulay) [inline services/core-web/src/components/Forms/incidents/IncidentForm.js:81]: A switch statement might have worked well here.

File: services/core-web/src/components/Forms/incidents/IncidentForm.js [hunk 3]
```diff
@@ -73,6 +75,37 @@ const documentColumns = [
...
+  } else if (status === "IMS") {
+    text = `This incident is missing critical information. The proponent has been notified that further information is required.`;
+  } else if (status === "AFR") {
+    text = `The incident was determined to be a dangerous occurence by the reporter and requires a final report to be submitted.`;
+  } else if (status === "INV") {
+    text = `This incident currently an EMLI under investigation.`;
+  } else if (status === "MIU") {
+    text = `This incident currently under MIU investigation.`;
+  } else if (status === "FRS") {
+    text = `This incident contains a final report submitted.`;
```
Comment (JustinMacaulay) [inline services/core-web/src/components/Forms/incidents/IncidentForm.js]: Not sure this text makes sense.

**Review by JustinMacaulay — COMMENTED** [review]
Just a couple comments.

File: services/core-web/src/components/Forms/incidents/IncidentForm.js (hunk 3 above)
Comment (areyeslo) [reply services/core-web/src/components/Forms/incidents/IncidentForm.js]: I agree. text is coming from the designs

Comment (JustinMacaulay) [reply services/core-web/src/components/Forms/incidents/IncidentForm.js]: I would either re-word it or get it re-worded cause it is going to have to happen regardless

**Review by JustinMacaulay — DISMISSED** [review]
Looks good

**Review by BenHarrington — APPROVED** [review]

**Pull request merged**